### PR TITLE
[IMPROVEMENT] TwoFactor component styling

### PR DESCRIPTION
**Pull request by youssef-md**
## Proposed changes
This PR improves the TwoFactor component styling to enhance visual hierarchy.

## Issues
Depends on #2314 

## How to test or reproduce
Set 2fa in your account, and login again.

## Screenshots
Group 797 (1)

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] Improvement (non-breaking change which improves a current function)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update (if none of the other choices apply)

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/RocketChat/Rocket.Chat/blob/develop/.github/CONTRIBUTING.md#contributing-to-rocketchat) doc
- [x] I have signed the [CLA](https://cla-assistant.io/RocketChat/Rocket.Chat.ReactNative)
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)
- [ ] I have added necessary documentation (if applicable)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/containers/TwoFactor/styles.js [hunk 1]
```diff
@@ -9,12 +9,14 @@ export default StyleSheet.create({
 		alignItems: 'center'
 	},
 	content: {
-		padding: 16,
+		paddingHorizontal: 16,
+		paddingVertical: 25,
```
Comment (djorkaeffalexandre) [inline app/containers/TwoFactor/styles.js:13]: Always use multiples of 4

File: app/containers/TwoFactor/styles.js [hunk 2]
```diff
@@ -9,12 +9,14 @@ export default StyleSheet.create({
...
 		width: '100%',
 		borderRadius: 4
 	},
 	title: {
-		fontSize: 14,
+		fontSize: 16,
+		textAlign: 'center',
```
Comment (djorkaeffalexandre) [inline app/containers/TwoFactor/styles.js:19]: `sharedStyles.textAlignCenter`

File: app/containers/TwoFactor/styles.js [hunk 3]
```diff
@@ -32,7 +34,8 @@ export default StyleSheet.create({
 		...sharedStyles.textRegular
 	},
 	button: {
-		marginBottom: 0
+		marginBottom: 0,
+		width: '48%'
```
Comment (djorkaeffalexandre) [inline app/containers/TwoFactor/styles.js:38]: Looks weird, I guess the button don't need to cover the entire modal, this is a pattern also used on our UploadModal.
WDYT @diegolmello?

Comment (diegolmello) [reply app/containers/TwoFactor/styles.js:38]: Right.
I thought this PR would be only for the title btw.

Comment (youssef-md) [reply app/containers/TwoFactor/styles.js:38]: No problem!

Comment (youssef-md) [reply app/containers/TwoFactor/styles.js]: Done!

**Pull request merged**